core/backend.go

```diff
@@ -52,16 +52,17 @@ type Backend interface {
 	MessageConstructor
 	Verifier
 
-	// BuildProposal builds a new block proposal
-	BuildProposal(view *proto.View) []byte
+	// BuildBlock builds a new ethereum block
+	BuildBlock(height uint64) []byte
 
 	// InsertBlock inserts a proposal with the specified committed seals
-	InsertBlock(proposal []byte, committedSeals []*messages.CommittedSeal)
+	// the reason why we are including round here is because a single committedSeal has signed the tuple of (EB, r)
```

Comment:
Change `EB` to `rawData`?

```suggestion
	// the reason why we are including round here is because a single committedSeal has signed the tuple of (rawData, r)
```

Reply:
Fixed fce6b2fbdfee5c50d903a50773047bfc6536ea6b